project/plugins.sbt

```diff
@@ -2,4 +2,5 @@ addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.6")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.2")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.5")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
+addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.6")
```

Comment:
to generate the zip package of all jars

Reply:
```
sbt universal:packageBin
```
will create `target/universal/rikai-0.0.13-SNAPSHOT.zip`

To make it work with PySpark, we need to put the following two jars with PySpark.
```
org.bytedeco.ffmpeg-4.4-1.5.6-macosx-x86_64.jar
org.bytedeco.javacpp-1.5.6-macosx-x86_64.jar
```

Here is how I put these two jars to local PySpark:
```
cp *-macosx-x86_64.jar /Users/da/.pyenv/versions/3.8.10/envs/rikai-example/lib/python3.8/site-packages/pyspark/jars
```

Reply:
Are there fallback-able java version for ffmpeg? Like if users forgot / dont know to install these two native packages, they can still run the code, just slower?

Reply:
no fallback-able java version now